### DISCUSSION TITLE
Dynamic integration

### DISCUSSION THREAD

**Discussion by 31trainman** [akash-network/community#505] [Integrations]
Dynamic is a web 3 login service for multiple blockchains and web 3 services. I have reached out to them to open a dialogue about the potential to integrate with cloudmos for a more unified login experience across multiple platforms. I will post updates in this thread as I get new information.

Here's the website for anyone interested in taking a look https://www.dynamic.xyz

**Comment by rodri-r** [comment]
Hey, Cloudmos already has a well implemented login experience, (accepting login with Keplr, Keplr mobile, Leap wallet and Cosmostation), what differentiates Dynamic from current login options in Cloudmos?

In my opinion the current Cloudmos login options are perfect for Akash, as these wallets are focused on Cosmos chains,such as Akash Network.

**Reply by 31trainman** [reply]
Dynamic allows one to use one wallet to login to multiple services. And works across multiple platforms. Plus it can manage multiple blockchains meaning users can manage multiple different services.

**Comment by nick134-bit** [comment]
AFAIK cloudmos uses cosmos-kit for the wallet connection, which is imo the best open source wallet connector for the cosmos ecosystem. I dont think there is need for a multichain wallet connection, that might be more closed source, and is overkill for a single chain application like cloudmos.

**Reply by 31trainman** [reply]
We want to allow as much ease of use as possible. Plus with a multi wallet approach it allows for managing of multiple assets and platforms without the complexity of multiple wallets needed across different blockchains.
